rebar.config.script

```diff
@@ -2,21 +2,29 @@
 
 IsWindows = case os:type() of {win32, _} -> true; {_, _} -> false end,
 
+FilterRocksDbCheck = (IsWindows orelse os:getenv("DISABLE_ROCKSDB") == "1"),
```

Comment:
I've 2 propositions:
Adding REBAR prefix seems more appropriate as now it's too generic/gloabal. e.g. `REBAR_DISABLE_ROCKSDB`

If we can ditch the == "1" check, so that it's exists or not would also be nice I guess :)

Reply:
Changed.